apps/react-18-tests-v9/tsconfig.json

```
@@ -13,8 +13,14 @@
   "include": [],
   "files": [],
   "references": [
+    {
+      "path": "./tsconfig.lib.json"
```

first of all ! thank you, this is the pattern we gonna adopt also for apps!1

small change is needed though - apps will have `.app.json` 


```suggestion
      "path": "./tsconfig.app.json"
```